x-pack/plugins/security/public/management/api_keys/api_keys_grid/create_api_key_flyout.tsx

```diff
@@ -312,6 +327,49 @@ export const CreateApiKeyFlyout: FunctionComponent<CreateApiKeyFlyoutProps> = ({
             )}
           </EuiFormFieldset>
 
+          <EuiSpacer />
+          <EuiFormFieldset>
+            <EuiSwitch
+              id="apiKeyCustom"
```

Comment:
This id is a duplicate of the one used above to restrict the key's privileges. Can we rename this id to avoid the conflict?

https://github.com/elastic/kibana/blob/f6bddaeb81280783ab5b50e0e950201c78ac0e19/x-pack/plugins/security/public/management/api_keys/api_keys_grid/create_api_key_flyout.tsx#L244-L254